dockerfiles/ci/xfail_tests/5.5.list

```diff
@@ -77,6 +77,7 @@ ext/date/tests/date_interval_create_from_date_string_nullparam.phpt
 ext/date/tests/date_time_set_variation3.phpt
 ext/date/tests/date_timestamp_set_nullparam2.phpt
 ext/date/tests/test-parse-from-format.phpt
+ext/fileinfo/tests/bug68819_002.phpt
```

Comment:
This was added to due to this: https://github.com/php/php-src/commit/471540d2853b3077af5dfbbbf2146edc6228a0cb.